digital-credentials/get-user-activation.https.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Is this a bad merge?

Isn't this file added by https://github.com/web-platform-tests/wpt/pull/46642

Reply:
oh yes, sorry... I copied too many files over. 🙈